### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Benteth**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://openjsf.org/cla) (after creating PR)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [x] Added/Updated related tests.

### CODE REVIEW

**Comment by Benteth** [comment]
> 
> 
> 
> 
> ## Pull request checklist
> 
> 
> 
> Make sure you:
> 
> 
> 
> - [x] Signed the [Contributor License Agreement](https://openjsf.org/cla) (after creating PR)
> 
> - [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)
> 
> 
> 
> For non-trivial changes, please make sure you also:
> 
> 
> 
> - [x] Added/Updated related documentation.
> 
> - [x] Added/Updated related tests.
> 
> 
> 
> ## Short description of the change(s)
> 
> 
> 
> 
> 
>

**Comment by captainbrosset** [comment]
Closing this PR as I don't understand what it does and seems to be incorrect.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ before contributing.
 
 <!-- Link labels -->
 
-[cla]: https://cla.js.foundation/webhintio/hint
+[cla]: https://openjsf.org/cla
```
Comment (Benteth) [inline .github/CONTRIBUTING.md:26]:
```suggestion
[cla]: https://openjsf.org/cla
```

**Review by Benteth — COMMENTED** [review]
@Benteth

**Pull request closed without merging**